tools/waveica/waveica_wrapper.R

```diff
@@ -38,7 +38,7 @@ Check that the following columns are present in your dataset: [sampleName, class
         alpha = alpha
         )
 
-    data[, -c(1:5)] <- features
+    data[, -c(1:5)] <- features$data_wave
```

Comment:
I think it would be more robust to include and exclude the columns by name instead of by index.

Reply:
I agree, fixed in 2413166